test/metabase/notification/send_test.clj

```diff
@@ -118,3 +119,61 @@
                                                                                    [:timestamp :string]
                                                                                    [:message :string]]])}}
                       (t2/select-one :model/TaskHistory :task "channel-send"))))))))))
+
+(deftest send-notification-record-prometheus-metrics-test
+  (mt/with-prometheus-system! [_ system]
+    (notification.tu/with-notification-testing-setup
+      (mt/with-temp [:model/Channel chn notification.tu/default-can-connect-channel]
+        (let [n (models.notification/create-notification!
+                 {:payload_type :notification/testing}
+                 nil
+                 [{:channel_type notification.tu/test-channel-type
+                   :channel_id   (:id chn)
+                   :recipients   [{:type :notification-recipient/user :user_id (mt/user->id :crowberto)}]}])
+              original-render @#'channel/render-notification]
+          (with-redefs [channel/render-notification (fn [& args]
+                                                      (is (prometheus-test/approx= 1 (mt/metric-value system :metabase-notification/concurrent-tasks {:payload-type "notification/testing"})))
```

Comment:
How about testing that this metric goes back down to 0 later in the test?

Reply:
added!